### PR TITLE
[Issue #96]Temporary solution to make travis fail

### PR DESCRIPTION
**Pull request by isabelrios**

### CODE REVIEW

**Comment by isabelrios** [comment]
I know the best option would be to change the TEAM test suite directly but it will require more time. In the meantime I worked on this solution to have the OGC test suite failing when tests fail and also to have two separate jobs, one for API tests and the other allowing failures for OGC tests so that travis is not always red till the OGC issues found are fixed.

@ferjm what do you think? thanks!

File: .travis.yml [hunk 1]
```diff
@@ -15,8 +22,8 @@ before_script:
 - npm run test
 - npm run example &
```
Comment (ferjm) [inline .travis.yml]: We don't need to run the example for the API tests job.

File: .travis.yml [hunk 2]
```diff
@@ -15,8 +22,8 @@ before_script:
 - npm run test
```
Comment (ferjm) [inline .travis.yml]: No need for `npm run test` for the OGC tests job.

Comment (isabelrios) [inline .travis.yml]: This is now only for API tests

File: .travis.yml (hunk 1 above)
Comment (isabelrios) [inline .travis.yml]: I do not get this command working only for OGC tests...
I tried the same way than the other being different with the & is giving errors
if [[ "$TESTS" = "OGC_TESTS" ]]; then npm run example & ; fi

But The command "if [[ "$TESTS" = "OGC_TESTS" ]]; then npm run example & ; fi" failed and exited with 1

Trying to get a solution, any idea @ferjm?
Thanks!

Comment (isabelrios) [inline .travis.yml]: Finally got that command working

**Comment by isabelrios** [comment]
Comments addressed and travis green, merging

**Pull request merged**